### PR TITLE
Reorganize README

### LINKED ISSUE #537: Mention extensions in the README?
So far I think  `polarssql` and `tidypolars`  are the only extensions based on `polars`

### PR DESCRIPTION
**Pull request by etiennebacher**
Followup of #602, close #537

### CODE REVIEW

**Comment by etiennebacher** [comment]
@grantmcdermott new PR here if you want to take a look

**Review by eitsupi — APPROVED** [review]
Thanks!

File: README.Rmd [hunk 1]
```diff
@@ -1,7 +1,5 @@
 ---
-output:
-  github_document:
-    html_preview: false
+output: github_document
```
Comment (eitsupi) [inline README.Rmd:2]: If I remember correctly, won't this change create an extra HTML?

Comment (etiennebacher) [reply README.Rmd:2]: Rendering it locally doesn't create an HTML file

**Comment by grantmcdermott** [comment]
LGTM

File: README.Rmd (hunk 1 above)
Comment (eitsupi) [reply README.Rmd:2]: I render it and a html file was generated.

Comment (etiennebacher) [reply README.Rmd:2]: My bad, I'll fix this

**Pull request merged**